web/src/components/repo/pipeline/PipelineItem.vue

```diff
@@ -54,7 +54,7 @@
 
         <div class="flex min-w-0 items-center space-x-2">
           <Icon name="commit" />
-          <span class="truncate">{{ pipeline.commit.slice(0, 10) }}</span>
+          <span class="truncate">{{ pipeline.commit_pipeline.sha.slice(0, 10) }}</span>
```

Comment:
As the api is now backwards compatible, I would suggest to extract these changes to a followup PR, so we can check such fundamental changes one-by-one.

Reply:
Would favor to do this in small steps as it would allow us to deploy this first and check if the api is still working as before and later on we could check the ui update using the new data fields.